src/main.rs

```diff
@@ -31,7 +31,6 @@
 
 use clap::{crate_version, App, Arg, ArgMatches, SubCommand};
 use image::Luma;
-use qr_code;
```

Comment:
was that un unnecessary import, then? my bad!

Reply:
yes! in 2018+ edition all external crates are implicitly imported into the namespace